docs/pages/versions/unversioned/guides/setup-native-firebase.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Setup Native Firebase
+---
+
+Because the native Firebase SDK requires build-time configuration, you cannot use it with the managed Expo client from the App Store. Luckily there are a few workarounds that we'll cover in this guide.
+
+- **Bare-Workflow (Universal):** You can add any native packages you want, but you cannot currently build a bare-workflow project in the cloud with Expo's build service.
+- **Custom-Workflow (iOS Only):** Create a custom Expo client in the cloud and gain access to the few Firebase packages that are in Expo (this does not include react-native-firebase).
```

Comment:
this is still called managed workflow, best not to introduce a new term in this doc